sencha-workspace/SlateTasksTeacher/app/controller/Viewport.js

```diff
@@ -1,34 +0,0 @@
-Ext.define('SlateTasksTeacher.controller.Viewport', {
-    extend: 'Ext.app.Controller',
-    requires: [
-        'Slate.API'
-    ],
-    
-    
-    config: {
-    },
-    
-    
-    // controller configuration
-    views: [
-        'Dashboard'
```

Comment:
@aliyeysides Dashboard view require again going missing and generating a warning